index.js

```diff
@@ -72,7 +72,6 @@ module.exports = async ({
     db: apiDB,
     port: wsServerPort,
     algoServerURL: `http://localhost:${algoServerPort}`,
-    hfDSBitfinexURL: `http://localhost:${hfDSBitfinexPort}`,
```

Comment:
should we remove `hfBitfinexDBPath` and `hfDSBitfinexPort` as well ? I don't think we are using them anyway..

Reply:
`hfBitfinexDBPath` is used for a conditional spawn of the data-server, which is not related to this PR which removes the data server client.

would prefer to make the change and test it in a seperate PR